### PR TITLE
Enable/Disable the Spheres in the scene (Benchmark app)

### PR DESCRIPTION
**Pull request by elviscapiaq**
- It also changes the visibility of knobs related to the spheres accordingly.

### CODE REVIEW

**Review by wangra-google — APPROVED** [review]

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 1]
```diff
@@ -659,8 +679,12 @@ void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainP
 
     // Record commands for the scene using one renderpass
     frame.cmd->BeginRenderPass(currentRenderPass);
```
Comment (angela28chen) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:681]: If both the skybox and the sphere are disabled, I think this renderpass should not be begin/ended?

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 2]
```diff
@@ -53,6 +53,22 @@ void GraphicsBenchmarkApp::InitKnobs()
     pDrawCallCount->SetDisplayName("DrawCall Count");
     pDrawCallCount->SetFlagDescription("Select the number of draw calls to be used to draw the `sphere-count` spheres.");
 
+    pAlphaBlend = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("alpha-blend", false);
```
Comment (angela28chen) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:56]: Can you also change the `Indent` of all the knobs related to the spheres to 1? Similar to how the quads related knobs are indented and displayed below the `pFullscreenQuadsCount` knob.

**Review by angela28chen — CHANGES_REQUESTED** [review]

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 3]
```diff
@@ -659,8 +679,12 @@ void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainP
 
     // Record commands for the scene using one renderpass
     frame.cmd->BeginRenderPass(currentRenderPass);
-    RecordCommandBufferSkybox(frame);
-    RecordCommandBufferSpheres(frame);
+    if (pEnableSkyBox->GetValue()) {
```
Comment (wangra-google) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:682]: we need to disable the `currentRenderPass` when both skybox and spheres are disabled?

**Review by wangra-google — CHANGES_REQUESTED** [review]

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 3 above)
Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:682]: Done.

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 2 above)
Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:56]: Done.

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 3 above)
Comment (wangra-google) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:682]: sorry i couldn't find this change to disable the render pass when both both skybox and spheres are disabled, is it in a different PR?

Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:682]: Sorry, when I rebased this PR with the last commits, I realized the Enable/Disable knobs for Skybox and Spheres were not working as it used to be and it seems the issue is related to recent commits. But now they were fixed with https://github.com/google/bigwheels/pull/331, so I will wait to this to be merged to update.

Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp]: Updated!

**Comment by elviscapiaq** [comment]
I fixed the issues with the render passes: when nothing is rendered, then the renderpass for GUI should be cleared (ATTACHMENT_LOAD_OP_CLEAR) and also I rebased it.

**Review by wangra-google — APPROVED** [review]

**Comment by elviscapiaq** [comment]
I rebased it with the main branch.

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 4]
```diff
@@ -25,13 +25,23 @@ void GraphicsBenchmarkApp::InitKnobs()
...
     pKnobVs->SetDisplayName("Vertex Shader");
     pKnobVs->SetFlagDescription("Select the vertex shader for the graphics pipeline.");
+    pKnobVs->SetIndent(1);
 
     pKnobPs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("ps", 0, kAvailablePsShaders);
     pKnobPs->SetDisplayName("Pixel Shader");
     pKnobPs->SetFlagDescription("Select the pixel shader for the graphics pipeline.");
+    pKnobPs->SetIndent(1);
 
     pAllTexturesTo1x1 = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("all-textures-to-1x1", false);
```
Comment (angela28chen) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:46]: Should be indent of 2 now

**Review by angela28chen — APPROVED** [review]
LGTM, thanks for incorporating the changes for new knob pAllTexturesTo1x1

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 4 above)
Comment (angela28chen) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:46]: Should be indent of 2 (sorry, accidentally deleted this comment earlier)

Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:46]: Done.

**Comment by elviscapiaq** [comment]
I rebased the PR with the `open_xr` support for the Benchmark app.

**Comment by elviscapiaq** [comment]
Rebased it and ready to be merged.

**Pull request merged**